vars/buildDockerImage.groovy

```diff
@@ -12,6 +12,12 @@ void call(Map args = [:]) {
     def build_qualifier = inputManifest.build.qualifier
     def build_number = args.buildNumber ?: "${BUILD_NUMBER}"
 
+    String image_tag =""
+
+    if (args.buildDate != null){
```

Comment:
Since you introduce new build args, can you please update this lib to have all the args and details in comments? 
Thanks.

Reply:
Sure Peter